.github/workflows/pull_request.yml

```diff
@@ -9,6 +9,7 @@ env:
     PYCURL_SSL_LIBRARY: openssl
     ROBOTTELO_BUGZILLA__API_KEY: ${{ secrets.BUGZILLA_KEY }}
     ROBOTTELO_JIRA__API_KEY: ${{ secrets.JIRA_KEY }}
+    ROBOTTELO_ROBOTTELO__unit_test: true
```

Comment:
`skip_dynaconf_validation` or similar with shorter text would make more sense to me. Isn't it should be in CAPS ?

Reply:
caps for consistency

Reply:
Updated.  The new name is `SKIP_VALIDATION` and it is under `robottelo.SETTINGS`.